es/mappings/make.py

```diff
@@ -342,6 +342,14 @@ def IsCyrillic(lang, something):
                 "rank": {
                     "type": "integer",
                 },
+
+                "execution_time_log": {
```

Comment:
This has to be nested, otherwise we will not be able to associate time with operation:
https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html